playwright/utils/generateRandomStringOfLength.ts

```diff
@@ -0,0 +1,7 @@
+export function generateRandomStringOfLength(length: number): string {
```

Comment:
```suggestion
export function getRandomString({ length }: { length: number }): string {
```

Reply:
Good idea with shortening the name, it's still clear what is it for.
But I don't understand what is the benefit of wrapping length in an object as a parameter?

Reply:
not important, but two benefits:
- the caller side will be `getRandomString({ length: 32 })` instead of `getRandomString(32)`.
- if tomorow you need to add another param, you add it to the object

Reply:
Ok, make a lot of sense!